tools/src/versioning/ios/index.ts

```diff
@@ -804,10 +826,13 @@ if pod_name.start_with?('${versionedPodNames.React}') || pod_name == '${versione
   target_installation_result.native_target.build_configurations.each do |config|
     config.build_settings['OTHER_CFLAGS'] = %w[
       ${configValues.join(`\n${indent}`)}
+      -fmodule-map-file="\${PODS_ROOT}/Headers/Public/${versionName}React-Core/${versionName}React/${versionName}React-Core.modulemap"
+      -fmodule-map-file="\${PODS_ROOT}/Headers/Public/${versionName}ExpoModulesCore/${versionName}ExpoModulesCore.modulemap"
```

Comment:
since we overrode the `OTHER_CFLAGS` in postinstall, we then miss the `-fmodule-map-file` generated by cocoapods. without this, we cannot find module like `@import ABI49_0_0React;`. here to add React and ExpoModulesCore back.